rust/parquet/src/reader/schema.rs

```diff
@@ -72,6 +73,18 @@ where
         .map(|fields| Schema::new(fields))
 }
 
+/// Convert parquet column schema to arrow field.
+pub fn parquet_to_arrow_field(parquet_column: ColumnDescPtr) -> Result<Field> {
+    let schema = parquet_column.self_type_ptr();
+
+    let mut leaves = HashSet::new();
+    leaves.insert(parquet_column.self_type() as *const Type);
+
+    ParquetTypeConverter::new(schema, Rc::new(leaves))
+        .to_field()
+        .map(|opt| opt.unwrap())
```

Comment:
Can this unwrap cause a panic?

Reply:
This should not happen. If it's empty, it must be a bug.